beetmoverscript/templates/fennec_nightly_repacks_signed.yml

```diff
@@ -3,15 +3,11 @@
 mapping:
   {{ locale }}:
     # common deliverables
-    apk:
-      artifact: target.apk
+    target.apk:
       s3_key: fennec-{{ version }}.{{ locale }}.android-arm.apk
       update_balrog_manifest: True
-    # TODO: to add checksums once we enable COT
-    #checksums:
-      #artifact: target.checksums
+    # Bug 1316124 - figure out checksums
+    #target.checksums:
```

Comment:
As above, unrelated to the code change, but a reminder for us all to clarify this before Tier-1 (bug 1316124)